release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2745,6 +2745,40 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.15.36
+[id="ocp-4-15-36_{context}"]
+=== RHSA-2024:7594 - {product-title} 4.15.36 bug fix and security update
+
+Issued: 09 October 2024
+
+{product-title} release 4.15.36, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:7594[RHSA-2024:7594] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHBA-2024:7597[RHBA-2024:7597] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.36 --pullspecs
+----
+
+[id="ocp-4-15-36-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, if a connection on port 9637 for Windows nodes was refused, the Kubelet Service Monitor threw a `target down` alert because CRIO doesn't run on Windows nodes. With this release, Windows nodes are excluded from the Kubelet Service Monitor. (link:https://issues.redhat.com/browse/OCPBUGS-42586[*OCPBUGS-42586*])
+
+* Previously, a change in the ordering of the `TextInput` parameters for PatternFly v4 and v5 caused the `until` field to be improperly filled and you could not edit it. With this release, the `until` field is editable so you can input the correct information. (link:https://issues.redhat.com/browse/OCPBUGS-42834[*OCPBUGS-42384*])
+
+* Previously, when the Node Tuning Operator (NTO) was configured using PerformanceProfiles, it created the `ocp-tuned-one-shot systemd` service. The service ran before Kubelet and blocked the service execution. The `systemd` service invoked Podman, which used the NTO image. If the NTO image was not present, Podman tried to fetch the image. 
++
+With this release, support is added for cluster-wide proxy environment variables that are defined in the `/etc/mco/proxy.env` environment. This allows Podman to pull NTO images in environments that need the HTTP/HTTPS  proxy for out-of-cluster connections. (link:https://issues.redhat.com/browse/OCPBUGS-42284[*OCPBUGS-42284*])
+
+* Previously, a node registration issue prevented you from using Redfish Virtual Media to add an xFusion bare metal node to your cluster. The issue occurred because the hardware was not fully compliant with Redfish. With this release, you can add xFusion bare-metal nodes to your cluster. (link:https://issues.redhat.com/browse/OCPBUGS-38798[*OCPBUGS-38798*])
```

Comment:
```suggestion
* Previously, a node registration issue prevented you from using Redfish Virtual Media to add an xFusion bare-metal node to your cluster. The issue occurred because the hardware was not fully compliant with Redfish. With this release, you can add xFusion bare-metal nodes to your cluster. (link:https://issues.redhat.com/browse/OCPBUGS-38798[*OCPBUGS-38798*])
```
See SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_b

Reply:
thank you! good catch!